spec/mailers/answer_mailer_spec.rb

```diff
@@ -9,11 +9,9 @@
   let(:sender_email) { 'innovation.forpost@gmail.com' }
 
   before do
-    # rubocop:disable Style/OpenStructUse
```

Comment:
Removed unnecessary disabling of Style/OpenStructUse. According to the recommendation of rubocop.

Reply:
haha interesting, when I was working on it - it was raising an error without it